lib/MusicBrainz/Server/Report/CatNoLooksLikeLabelCode.pm

```diff
@@ -0,0 +1,45 @@
+package MusicBrainz::Server::Report::CatNoLooksLikeLabelCode;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub component_name { 'CatNoLooksLikeLabelCode' }
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id, rl.catalog_number,
+            row_number() OVER (ORDER BY ac.name COLLATE musicbrainz, r.name COLLATE musicbrainz)
+        FROM
+            release_label rl
+            JOIN release r
+            ON r.id = rl.release
+            JOIN artist_credit ac ON r.artist_credit = ac.id
+        WHERE rl.catalog_number ~ '^LC([\s]+)?[0-9]{4,5}\$'
```

Comment:
Group `()` is unneeded, and `-` should probably be matched too as there is one `LC - 03090` for example:
```suggestion
        WHERE rl.catalog_number ~ '^LC[\s-]*[0-9]{4,5}\$'
```